recipe/patches/pre-7.3.6-0008.patch

```diff
@@ -17,8 +17,8 @@ diff -r 77787b8f4c49 -r fc8388ffecc6 lib-python/3/venv/__init__.py
                  src_library = os.path.join(os.path.dirname(context.executable),
                                             libname)
 diff -r 77787b8f4c49 -r fc8388ffecc6 pypy/tool/release/package.py
---- a/pypy/tool/release/package.py	Mon Apr 12 07:11:48 2021 +0200
-+++ b/pypy/tool/release/package.py	Sun Jul 11 12:46:37 2021 +0300
+--- a/pypy/tool/release/package.py	2021-4-12 07:11:48.000 +0200
++++ b/pypy/tool/release/package.py	Sun Jul 11 12:46:37.000 2021 +0300
```

Comment:
Aside from the fact that git (& `patch`) should easily handle the other dates as well, this looks inconsistent (now a mix of both formats). In any case, the patch seems to have been applied without issue:
```
Patch analysis gives:
[[ RA--D1L-VE ]] - [[                                                 patches/fficurses.patch ]]
[[ RA--D1LO-E ]] - [[                                               patches/tklib_build.patch ]]
[[ RA--D1LOVE ]] - [[                                                    patches/tk_app.patch ]]
[[ RA--D1L-VE ]] - [[                                                patches/lzma_build.patch ]]
[[ RA-MD1LOVE ]] - [[                                  patches/site-and-sysconfig-conda.patch ]]
[[ RA-MD1LOVE ]] - [[                                         patches/distutils-install.patch ]]
[[ RA-MD1LOVE ]] - [[                        patches/distutils-sysconfig-get_python_lib.patch ]]
[[ RA-MD1L-VE ]] - [[ patches/0009-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch ]]
[[ RA-MD1LOVE ]] - [[ patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch ]]
[[ RA-MD1L-VE ]] - [[                patches/0013-Disable-new-dtags-in-unixccompiler.py.patch ]]
[[ RA-MD1L-VE ]] - [[           patches/0020-Use-ranlib-from-env-if-env-variable-is-set.patch ]]
[[ RA-MD1L-VE ]] - [[                                            patches/pre-7.3.6-0001.patch ]]
[[ RA-MD1LOVE ]] - [[                                            patches/pre-7.3.6-0002.patch ]]
[[ RA-MD1L-VE ]] - [[                                            patches/pre-7.3.6-0003.patch ]]
[[ RA-MD1L-VE ]] - [[                                            patches/pre-7.3.6-0004.patch ]]
[[ RA-MD1LOVE ]] - [[                                            patches/pre-7.3.6-0005.patch ]]
[[ RA-MD1L-VE ]] - [[                                            patches/pre-7.3.6-0006.patch ]]
[[ RA-MD1L-VE ]] - [[                                            patches/pre-7.3.6-0007.patch ]]
[[ RA-MD1L--E ]] - [[                                            patches/pre-7.3.6-0008.patch ]]
```

Reply:
yeah, I got confused by a failure at some point and tried to modify this by hand. I think patch ignores the date altogether, so it shouldn't matter